exonum/src/sandbox/sandbox.rs

```diff
@@ -268,6 +267,26 @@ impl Sandbox {
         }
     }
 
+    pub fn send_peers_request(&self) {
+        self.process_events();
+        let send = self.inner.borrow_mut().sent.pop_front();
+
+        if let Some((addr, msg)) = send {
+            let peers_request =
+                PeersRequest::from_raw(msg).expect("Incorrect message. PeersRequest was expected");
+
+            let id = self.addresses.iter().position(|&a| a == addr);
+            if let Some(id) = id {
+                assert_eq!(&self.p(ValidatorId(id as u16)), peers_request.to());
+            } else {
+                panic!("Sending PeersRequest to unknown peer {:?}", addr);
+            }
+            trace!("Sending {:?}", peers_request);
```

Comment:
Why do we need logging in sandbox?

Reply:
That wasn't my idea. If you grep - you'll find another traces in the sandbox' source and I don't think it's bad.
Could you please explain why you think this is "wrong"?

Reply:
Oh, actually, it's bad idea to use `trace` level. `info` should be better.

Reply:
@vitvakatu should I also raise logging level to `info` in `sandbox_tests_helper.rs`?

Reply:
In that particular case -- we have either `panic` (and `trace` will not be reached) or `trace`. Thus `trace` will be shown only if test is passed, and logging, why test did pass is a strange thing -- it should pass because it is expected to pass, isn't it?

Reply:
I don't log "why" test passed. I log what's going on in this particular moment. Just to be able to see that not in code but printed in the console. 
It won't affect **anything** so I don't understand why it worths discussion at all.

Reply:
I don't insist on anything, I just don't see any profit in this line. However, it does cost some CPU ticks, and if it doesn't help us (and I still think that it doesn't), I suppose we should not include it in the code.
Once again, it's no more than a question, if you're sure it's a useful thing, feel free to ignore my nit :)

Reply:
Just to elaborate - I don't think `trace` is a good idea because log output will be heavily mixed with a lot of log entries from rustc and cargo.

Reply:
@vitvakatu at least one can do `RUST_LOG=exonum=trace`, so I don't consider that a problem at all.

Reply:
I found some method names in the sandbox confusing. (I am not talking about this PR).
For example here `trace!("Sending {:?}", peers_request);` make me think that this method is responsible for sending some messages, but in reality, messages are sent by NodeHandler and this function just asserts that this message was sent.
For instance, in the file node/basic.rs there is already tracing for this message  `trace!("Request peers from peer with addr {:?}", peer.addr());`
Please correct me if I am wrong.

Reply:
> I found some method names in the sandbox confusing.
I found all of them confusing.